modules/Bio/EnsEMBL/DBSQL/TranslationAdaptor.pm

```diff
@@ -965,4 +965,18 @@ sub fetch_all {
   return \@translations;
 }
 
+
+# _tables
```

Comment:
Strictly speaking this should be a =head2 block of pod in order for it render in the API reference. It is however an internal method so it can pass.